tests/lib/rules/no-dom-import.test.ts

```diff
@@ -4,29 +4,50 @@ import { createRuleTester } from '../test-utils';
 const ruleTester = createRuleTester();
 
 const SUPPORTED_TESTING_FRAMEWORKS = [
-  'react-testing-library',
-  '@testing-library/react',
-  '@marko/testing-library',
+  {
+    configOption: 'angular',
+    oldName: '@testing-library/angular',
+    newName: '@testing-library/angular',
+  },
+  {
+    configOption: 'react',
+    oldName: 'react-testing-library',
+    newName: '@testing-library/react',
+  },
+  {
+    configOption: 'vue',
+    oldName: 'vue-testing-library',
+    newName: '@testing-library/vue',
+  },
+  {
+    configOption: 'marko',
+    oldName: '@marko/testing-library',
+    newName: '@marko/testing-library',
+  },
 ];
 
 ruleTester.run(RULE_NAME, rule, {
   valid: [
     'import { foo } from "foo"',
     'import "foo"',
-    ...SUPPORTED_TESTING_FRAMEWORKS.flatMap((testingFramework) => [
-      `import { fireEvent } from "${testingFramework}"`,
-      `import * as testing from "${testingFramework}"`,
-      `import "${testingFramework}"`,
-    ]),
+    ...SUPPORTED_TESTING_FRAMEWORKS.flatMap(({ oldName, newName }) =>
+      [oldName, newName].flatMap((testingFramework) => [
```

Comment:
I don't quite understand the purpose of the `oldName` and `newName` from supported testing frameworks.

Reply:
`oldName` is like `react-testing-library`, so before the packages were under the `@testing-library` scope.